package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/smart-payment-buttons",
-  "version": "2.0.360",
+  "version": "2.0.361-4",
```

Could we back out the version number change from this PR? We manage the version number in the release process.
```suggestion
  "version": "2.0.360",
```